### DISCUSSION TITLE
Documentation missing on Landing Times view

### DISCUSSION THREAD

**Discussion by govindrai** [apache/airflow#25693] [General]
### What do you see as an issue?

Searching the docs for Landing Times returns no results though it is a primary view when looking at DAG info. The [UI/Screenshots](https://airflow.apache.org/docs/apache-airflow/stable/ui.html#) section explains most views but skips over landing times. 

### Solving the problem

I would like to understand the view/metric myself. Could we get an official explanation? Thanks! 

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
This is a good question - as a maintainer I have not looked at it, but when I queried google the first answer was https://stackoverflow.com/questions/37714261/what-does-the-landing-time-mean-in-airflow - i think it's pretty good and you can deep-dive and I would definitely encourage you to (once you get the hang of it) attempt to make a PR to our documentation and even contribute screenshot. There is no "official" explanation. Airflowi is created by > 2100 contributors and anyone - literally anyone can contribute a PR, code, documentation to make things clearer, better. And users who struggle with something, make research and find their ways are absolutely the best to describe such missing pieces of documentation - because maintainers have too many assumptions in their head.

So once you follow the search and find your answers, I sincerely encourage you to contribute a PR with explaining what the landing times are.

**Reply by potiuk** [reply]
I think you are actually one of the best people in the world to do it - you have interest, you are courious to find the answer and when you come with fresh mind, there is a very good chance your description will be at the level of other people asking simillar questions.
